dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/MYFACES-4658.war/resources/WEB-INF/beans.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes -- otherwise I get this error: 

 jakarta.servlet.ServletException: Cannot invoke "jakarta.faces.component.behavior.ClientBehavior.getScript(jakarta.faces.component.behavior.ClientBehaviorContext)" because the return value of "org.apache.myfaces.cdi.wrapper.FacesClientBehaviorCDIWrapper.getWrapped()" is null

Reply:
I created https://github.com/OpenLiberty/open-liberty/issues/28314 to follow up later. Not what is the proper behavior is here.